pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -2087,4 +2098,318 @@ var _ = Describe("v1beta1 inference service controller", func() {
 			}))
 		})
 	})
+	Context("Set CaBundle ConfigMap in inferenceservice-config confimap", func() {
+		It("Should not create a global cabundle configMap in a user namespace when CaBundleConfigMapName set ''", func() {
+			// Create configmap
+			var configMap = &v1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      constants.InferenceServiceConfigMapName,
+					Namespace: constants.KServeNamespace,
+				},
+				Data: configs,
+			}
+
+			Expect(k8sClient.Create(context.TODO(), configMap)).NotTo(HaveOccurred())
+			defer k8sClient.Delete(context.TODO(), configMap)
+
+			By("By creating a new InferenceService")
+			serviceName := "sample-isvc"
+			var expectedRequest = reconcile.Request{NamespacedName: types.NamespacedName{Name: serviceName, Namespace: "default"}}
+			var serviceKey = expectedRequest.NamespacedName
+			var storageUri = "s3://test/mnist/export"
+			ctx := context.Background()
+			isvc := &v1beta1.InferenceService{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      serviceKey.Name,
+					Namespace: serviceKey.Namespace,
+				},
+				Spec: v1beta1.InferenceServiceSpec{
+					Predictor: v1beta1.PredictorSpec{
+						ComponentExtensionSpec: v1beta1.ComponentExtensionSpec{
+							MinReplicas: v1beta1.GetIntReference(1),
+							MaxReplicas: 3,
+						},
+						Tensorflow: &v1beta1.TFServingSpec{
+							PredictorExtensionSpec: v1beta1.PredictorExtensionSpec{
+								StorageURI:     &storageUri,
+								RuntimeVersion: proto.String("1.14.0"),
+								Container: v1.Container{
+									Name:      constants.InferenceServiceContainerName,
+									Resources: defaultResource,
+								},
+							},
+						},
+					},
+				},
+			}
+			Expect(k8sClient.Create(ctx, isvc)).Should(Succeed())
+
+			caBundleConfigMap := &v1.ConfigMap{}
+			Eventually(func() bool {
```

Comment:
I think `Eventually` is fit in this place based on this definition.
```
 Eventually expects the condition to be met at some point in the future, while Consistently expects the condition to be consistently met over time. 
```
It does not matter when it can get the configmap. what we are looking for is at some point, it should be created and be readable.  WDYT?

Reply:
Unless I'm misunderstanding, the code is checking for absence of the ConfigMap.

To me, it can lead to false positives, because the tested condition is true since the beginning.

Changing to `Consistently` will ensure that it won't be created by mistake.

Reply:
oh right.  line #2403 should be `Eventually` but this one can be `Consistently`